python/sqlalchemy/src/config.py

```diff
@@ -0,0 +1,8 @@
+
+
+listen_port = 8080
+db_user = 'postgres'
+db_password = None
+database = 'postgres'
+db_host = 'localhost'
+db_port = 5433
```

Comment:
Whats the default log level? Would like this to be "info" level - higher than debug. Can we add that as a config constant here?

Reply:
Added default of logging.INFO